docs/source/deployment/databricks.md

```diff
@@ -169,10 +169,10 @@ In your newly-created notebook, put each of the below code snippets into a separ
 %sh rm -rf ~/projects/iris-databricks && git clone --single-branch --branch main https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com/${GITHUB_USER}/<your-repo-name>.git ~/projects/iris-databricks
 ```
 
-* Install the latest version of Kedro compatible with version `0.18.3`
+* Install the latest version of Kedro compatible with version `0.18.4`
 
 ```console
-%pip install "kedro[spark.SparkDataSet]~=0.18.3"
+%pip install "kedro[spark.SparkDataSet]~=0.18.4"
```

Comment:
I just remembered I need to add an issue to update the docs to point to `kedro-datasets[XXX]` after we release `1.0.0`.